crates/forge/src/cmd/build.rs

```diff
@@ -118,13 +118,13 @@ impl BuildArgs {
 
         // Only run the `SolidityLinter` if there are no compilation errors
         if !output.output().errors.iter().any(|e| e.is_error()) {
-            self.lint(&project, &config).map_err(|err| eyre!("Lint failed: {err}"))?;
+            self.lint(&project, &config, &self.paths).map_err(|err| eyre!("Lint failed: {err}"))?;
```

Comment:
can we not get the paths from the compiler output instead?

Reply:
@DaniPopes `artifacts_with_files` returns all artifacts including artifacts from deps while `input_files_iter` returns only src, test and scripts that we want to lint. Could use `artifacts_with_files` but then have to filter out deps artifacts - IMO is cleaner by `input_files_iter` but happy to change if you strongly think we should use `artifacts_with_files`. thank you!